installers/installer.sh

```diff
@@ -396,14 +396,15 @@ function __install_syncgateway_amazon() {
 }
 
 function __install_syncgateway_ubuntu() {
+    #https://packages.couchbase.com/releases/couchbase-sync-gateway/2.8.0/couchbase-sync-gateway-community_2.8.0_x86_64.deb
     local version=$1
     local tmp=$2
-    __log_info "Installing Couchbase Sync Gateway Enterprise Edition v${version}"
+    __log_info "Installing Couchbase Sync Gateway community Edition v${version}"
```

Comment:
@malscent HI Justin, hope all is well. Came across this and thought I could bootstrap this to be bale to deploy CC just like we are doing for EE in your script. Would you think this is a fine approach?

Reply:
Hey @sss329 Can you rebase with Main?  So i can see your specific changes?

Reply:
Hey @malscent sure give me one sec.

Reply:
Hey @malscent going to make another specific PR and point out changes one moment. Thanks btW!